egs/wham/wavesplit/model.py

```diff
@@ -0,0 +1,281 @@
+from torch import nn
+import torch
+from asteroid.masknn import norms
+from kmeans_pytorch import kmeans, kmeans_predict
+
+
+class Conv1DBlock(nn.Module):
```

Comment:
Don't you think we should make wavesplit part of asteroid not just the WHAM recipes ?